typing_tester.py

```diff
@@ -0,0 +1,137 @@
+import curses
+from curses import wrapper
+import time
+
+def cal_accuracy_score(given_word, user_text, no_of_backspaces):
+    """Calculates the accuracy score for typing tester"""
+
+    wrong_letter = 0
+    try:
+        wrong_letter = sum([1 for i in range(len(given_word)) if user_text[i] != given_word[i]])
+    except:
```

Comment:
Its recommended to catch specific type of exception for better readability, logic and it allows us to handle specific exception cases